### PR TITLE
Adding my script example to upload the E3SM-MMF dataset to LEAP's google cloud storage.

### PR DESCRIPTION
**Pull request by sungdukyu**

### CODE REVIEW

**Comment by jbusecke** [comment]
pre-commit.ci autofix

File: data-notebooks/example_sungduk.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+with open('path/to/authentication/file.json') as f:
+    token = json.load(f)
+    fs = gcsfs.GCSFileSystem(token=token)
+
+
+# Transfer data to GC
+# for j in ['train', 'test']:
+#    for k in ['input', 'output']:
+for j in ['train']:
+    for k in ['input']:
```
Comment (jbusecke) [inline data-notebooks/example_sungduk.py:65]: Is this the complete data? AFAICT this only produces one dataset `...train...input`?

**Review by jbusecke — COMMENTED** [review]
Thank you so much for submitting this PR. I noted a quick question in the review, but if you can confirm this works as intended, I will merge.

File: data-notebooks/example_sungduk.py (hunk 1 above)
Comment (sungdukyu) [reply data-notebooks/example_sungduk.py]: Thanks for catching that! I just fixed it at "9b6bd6c", which I just committed 2 hours ago.

**Comment by jbusecke** [comment]
Thanks @sungdukyu. Merging this now.

**Pull request merged**